wgsl/index.bs

```diff
@@ -5426,7 +5426,7 @@ ISSUE: Can we query upper bounds on workgroup size dimensions?  Is it independen
 
 <div class='example wgsl global-scope' heading='workgroup_size Attribute'>
   <xmp>
-    [[ stage(compute), workgroup_size(8,1,1) ]]
+    [[ stage(compute), workgroup_size(8,4u,1) ]]
```

Comment:
Not all components are the same type in this example

Reply:
Fixed.  Thanks.